monkeywagtail/artist/models.py

```diff
@@ -47,14 +47,20 @@ class Artist(ClusterableModel):
     The author snippet gives a way to add authors to a site and create a one-way relationship with content
     """
 
+    title = models.CharField("The artist's name", blank=True, max_length=254)
```

Comment:
If you have a field named "slug" it will automatically generate as you type into a field named "title". Since the admin JS already offers this functionality using these specific names I'd rather stick to convention than inject extra JS.